script.js

```diff
@@ -0,0 +1,191 @@
+const speakersList = document.querySelector('.speakers-list');
+const closeBtn = document.querySelector('.modal-close');
+const modalWindow = document.querySelector('.modal');
+const hamburger = document.querySelector('.hamburger');
+const home = document.querySelector('.nav-home');
+const about = document.querySelector('.nav-about-us');
+const aboutPage = document.querySelector('.about-us');
+const homePage = document.querySelector('.home');
+const activeLink = document.querySelector('.active');
+const logo = document.querySelector('.brand');
+
+const speakers = [
+  {
+    container: 'speaker',
+    speakerHolder: 'speaker-holder',
+    card: 'card',
+    backface: 'olara@gmail.com',
+    back: 'back',
+    imageDetails: ['images/olara.jpg', 'yori bonger\'s image', 'speaker-image'],
+    speakerBio: 'speaker-bio',
+    h3Class: 'speaker-name',
+    SpeakerName: 'Olara Peter',
+    pOneClass: 'profession',
+    pOneContent: 'Study masters in information technology at MakerereUniversity',
+    pTwoClass: 'role',
```

Comment:
Your app is not performing as expected. All pages, links, and images are not working as expected. The `speaker section` is not loading as expected. I got the error below after the app was loaded.

![image](https://user-images.githubusercontent.com/10332499/163654335-f84fb3fd-41f1-4f3b-bd8e-936153aee920.png)